admin/app/adapters/certification.js

```diff
@@ -21,8 +21,6 @@ export default class Certification extends ApplicationAdapter {
       const payload = {
         data: {
           attributes: {
-            'comment-for-organization': attributes['comment-for-organization'],
-            'comment-for-candidate': attributes['comment-for-candidate'],
```

Comment:
Je n'arrive pas à mettre de commentaire sur la ligne concernée (ligne 30).

Plus bas on effectue un `POST` sur l'url retorunée par la méthode `this.urlForUpdateComments()`.

On ne peut maintenant renvoyer qu'un seul commentaire. Renomme-t-on la méthode en `this.urlForUpdateComment` (sans le "s") ou en `this.urlForUpdateInternalJuryComment` ? 🤷‍♂️

Reply:
Tout à fait, on peut enlever la pluralité ici aussi, merci :)

Reply:
Si on veut aller au bout de l'idée, ça a pas mal d'impact 😁 
Je vous en parle tout à l'heure

Reply:
On traite dans une autre PR de refacto pour ne pas bloquer plus ce qui en dépend